specs/harness/ControllerHarness.sol

```diff
@@ -150,7 +150,7 @@ contract ControllerHarness is Controller {
   }
 
   function openVault(address owner, uint256 vaultId) external {
-    Actions.OpenVaultArgs memory args = Actions.OpenVaultArgs({owner: owner, vaultId: vaultId});
+    Actions.OpenVaultArgs memory args = Actions.OpenVaultArgs({owner: owner, vaultId: vaultId, vaultType: 0});
```

Comment:
Shouldn't this be dynamic somehow ?

Reply:
well for now just having this test on vaultType 0. For margin vaults, might need to write a new test. When I do that, will update this!